tools/rpkg/R/sql.R

```diff
@@ -0,0 +1,39 @@
+#' Run a SQL query
+#'
+#' `sql()` runs an arbitrary SQL query and returns a data.frame the query results
+#'
+#' @param sql A SQL string
+#' @param conn An optional connection, defaults to built-in default
+#' @return A data frame with the query result
+#' @export
+#' @examples
+#' print(sql("SELECT 42"))
+
+sql <- function(sql, conn = default_connection()) {
```

Comment:
This name will clash with dplyr/dbplyr.

Perhaps `duckdb_sql()` ?

Reply:
I'd rather have the shorthand in duckdb than not clash tbh